File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/UserNotFoundException.java
package com.dogGetDrunk.meetjyou.common.exception.business;

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode;

public class UserNotFoundException extends NotFoundException {

    private String userId;

    public UserNotFoundException(Long userId) {
        super(Long.toString(userId), ErrorCode.USER_NOT_FOUND);
    }

    public UserNotFoundException(String userId) {
        super(userId, ErrorCode.USER_NOT_FOUND);
    }
}

File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/UserNotFoundException.kt
package com.dogGetDrunk.meetjyou.common.exception.business

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode

class UserNotFoundException(
    val userId: String
) : NotFoundException(userId, ErrorCode.USER_NOT_FOUND) {
    constructor(userId: Long) : this(userId.toString())
}
